cppSettings.gradle

```diff
@@ -140,46 +140,58 @@ task clean(type: Delete) {
 }
 
 subprojects {
-    plugins.withType(CppPlugin).whenPluginAdded {
-        ext.defineWpiUtilProperties = {
-            ext.wpiUtil = wpiUtilUnzipLocation
-            ext.wpiUtilInclude = "$wpiUtilUnzipLocation/include"
-            ext.wpiUtilLibArmLocation = "$wpiUtilUnzipLocation/Linux/arm"
-            if (project.hasProperty('makeSim')) {
-                ext.wpiUtilLibDesktopLocation = "$wpiUtilUnzipLocation/Linux/amd64"
-            }
-            ext.wpiUtilStaticLib = "$wpiUtilLibArmLocation/libwpiutil.a"
+      ext.defineWpiUtilProperties = {
```

Comment:
Indentation is all wrong in this file. Our gradle files are indented with 4 spaces.